.github/labeler.yml

```diff
@@ -0,0 +1,48 @@
+# Config file for .github/workflows/labeler.yml to automatically add labels to
+# pull requests based on the globs below. For documentation see:
+# https://github.com/marketplace/actions/labeler
+---
+ci-cd:
```

Comment:
I'm not sure if `ci-cd` will match our label `ci/cd`

Reply:
Good spot, I'll make a PR to fix this.

Reply:
https://github.com/oneapi-src/unified-runtime/pull/1506